JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.html

```diff
@@ -19,27 +19,41 @@
   <div
     *dxTemplate="let dataCell of 'dataCellTemplate'"
     [ngClass]="{
-      'disable-date': isDisableDate(dataCell.startDate),
+      'disable-date':
+        (isHoliday | apply : dataCell.startDate : holidays) ||
+        (isWeekend | apply : dataCell.startDate),
       dinner:
-        !isDisableDate(dataCell.startDate) && isDinner(dataCell.startDate),
-      'dx-scheduler-date-table-cell-text': isMonthView()
+        !(
+          (isHoliday | apply : dataCell.startDate : holidays) ||
+          (isWeekend | apply : dataCell.startDate)
+        ) && (isDinner | apply : dataCell.startDate : dinnerTime),
```

Comment:
Let's change it (look at comment above):

```suggestion
        !(isDateDisabled | apply : dataCell.startDate) && (isDinner | apply : dataCell.startDate),
```